### DISCUSSION TITLE
How could we get the higlighted portion, quad (quad_like,rect_like) from page object ?

### DISCUSSION THREAD

**Discussion by Carnegie-eng** [pymupdf/PyMuPDF#2381] [Q&A] [answered]
I would like to get the position information on highlited portion in page of PDF document.

example:
import fitz
testPDF = fitz.open("PDF.pdf")
page = testPDF[0]

I can't look for the information on highlighted portion from page object above. Where does the informtion exited in the page object ? 
I appreciate if you could let me know !!

**Comment by JorjMcKie** [comment, marked as answer]
You can walk over the highlight annotations of the page, then take every of their rectangles and see what's underneath.
```python
for annot in page.annots(types=[fitz.PDF_ANNOT_HIGHLIGHT]):
    rect = annot.rect + (-5, -5, 5, 5)  # annot rect, 5 points larger in every direction
    print("highlighted:", page.get_textbox(rect))
```

**Reply by Carnegie-eng** [reply]
Thank you so much for your assistance ! 
I appreciate PyMuPDF package and your quick answer.
I am basically R user but PyMuPDF is only available in Python, so I include Python and PyMuPDF to R and web application. Thank you for wonderful package !

**Reply by JorjMcKie** [reply]
Thank you very much for that nice feedback!
